tests/test_files.py

```diff
@@ -68,12 +76,16 @@ def test_WellFile__opens_and_get_mantarray_serial_number():
     assert wf.get_mantarray_serial_number() == "M02001900"
 
 
-# def test_WellFile__opens_and_get_UTC_begin_recording():
-#     wf = WellFile(
-#         os.path.join(PATH_OF_CURRENT_FILE, "M120171010__2020_07_22_201922", "M120171010__2020_07_22_201922__A1.h5")
-#     )
+def test_WellFile__opens_and_get_begin_recording():
+    wf = WellFile(
+        os.path.join(
+            PATH_OF_CURRENT_FILE,
+            "2020_08_04_build_775",
+            "MA20001010__2020_08_04_220041__A1.h5",
+        )
+    )
 
-#     assert wf.get_UTC_begin_recording() == ""
+    assert wf.get_begin_recording() == "2020-08-04 22:01:27.491628"
```

Comment:
when get_begin_recording loads this metadata attribute, convert it to a python datetime object. makes comparisons and other functions easier down the road